decidim-admin/spec/system/admin_active_users_counter_spec.rb

```diff
@@ -12,9 +12,62 @@
     visit decidim_admin.root_path
   end
 
+  it "shows Welcome message" do
```

Comment:
I think we should rename this spec as it is no longer only about the active users count.

I would suggest something like `admin_checks_dashboard_counts_spec` and within the spec `describe "Admin checks dashboard panel counts"`.